squad/frontend/views.py

```diff
@@ -491,8 +491,9 @@ def metrics(request, group_slug, project_slug):
         "project": project,
         "environments": environments,
         "metrics": metrics,
-        "thresholds": list(project.metricthreshold_set.all().values(
-            'name', 'value')),
+        "thresholds": list(MetricThreshold.objects.filter(
```

Comment:
well it's been a while, but can you maybe pass thresholds as a dict of {'env1': [thresholds1]...} and then it'd be easier for you to classify them in the UI? just an idea...

Reply:
I think thresholds is passed to the charts controller `charts.js` and if i revamp the way it is passed to the UI, it might require more unnecessary code changes. AFAIK, as is it is working as it was originally.